assets/runtime/functions

```diff
@@ -584,6 +584,14 @@ gitlab_configure_analytics() {
   gitlab_configure_analytics_piwik
 }
 
+gitlab_configure_rack_attack() {
+  exec_as_git sid -i 's|{{RACK_ATTACK_ENABLED}}|'"${RACK_ATTACK_ENABLED}}"'|' ${GITLAB_INSTALL_DIR}/config/gitlab.yml
```

Comment:
typo in `sed` command name (all instances) :smile: 
can you fix and update PR. Also can you add `Configuring gitlab::rack_attack...` message like the other functions.

Please comment on the PR after you update it so that I am notified.